labman/db/process.py

```diff
@@ -3038,53 +3038,53 @@ def generate_prep_information(self):
                        run_name as run_prefix, sp.sequencer_id as platform_id,
                        sp.experiment as center_project_name
                 -- Retrieve sequencing information
-                FROM qiita.sequencing_process sp
-                LEFT JOIN qiita.equipment e ON (
+                FROM labman.sequencing_process sp
+                LEFT JOIN labman.equipment e ON (
                     sequencer_id = equipment_id)
-                LEFT JOIN qiita.equipment_type et ON (
+                LEFT JOIN labman.equipment_type et ON (
                     et.equipment_type_id = e.equipment_type_id)
-                LEFT JOIN qiita.sequencing_process_lanes spl USING (
+                LEFT JOIN labman.sequencing_process_lanes spl USING (
                     sequencing_process_id)
                 -- Retrieve pooling information
-                LEFT JOIN qiita.pool_composition_components pcc1 ON (
+                LEFT JOIN labman.pool_composition_components pcc1 ON (
                     pcc1.output_pool_composition_id = spl.pool_composition_id)
-                LEFT JOIN qiita.pool_composition pccon ON (
+                LEFT JOIN labman.pool_composition pccon ON (
                     pcc1.input_composition_id = pccon.composition_id)
-                 LEFT JOIN qiita.pool_composition_components pcc2 ON (
+                 LEFT JOIN labman.pool_composition_components pcc2 ON (
                     pccon.pool_composition_id =
                     pcc2.output_pool_composition_id)
                 -- Retrieve amplicon library prep information
-                LEFT JOIN qiita.library_prep_16S_composition lp ON (
+                LEFT JOIN labman.library_prep_16s_composition lp ON (
```

Comment:
Capitalization change here

Reply:
As you know, tried to fix this: https://github.com/jdereus/labman/issues/279 here but it was getting too confusing due to number of changes and travis errors; thus, decided to open an issue about it.

Reply:
:+1: